web/css-part-2.md

```diff
@@ -0,0 +1,94 @@
+# CSS: Part 2
+
+### Projected Time
+
+About 5 hours
+
+- Guided Practice: 90 min
+- 3.5 hours for Independent Practice (Project)
+- Check For Understanding: 10 min
+
+### Prerequisites
+
+- [HTML lesson](/web/html.md)
+- [Intro to CSS](./css.md)
+
+### Motivation
+
+- Cascading style sheets(CSS) is a style sheet language used for describing the look and formatting of a document written in a markup language like HTML. (referenced from [wikipedia](https://en.wikipedia.org/wiki/Cascading_Style_Sheets) & [town open-data portal](https://manualzz.com/doc/6479032/town-open-data-portal---dip%C3%B2sit-digital-de-la-ub))
+
+
+### Objectives
+
+**Participants will be able to:**
+
+
+
+### Specific Things to ~Learn~ Practice
+
+- CSS selectors
+- Common CSS properties and their allowable values
+- How to incorporate CSS into web projects
+- Web project file tree
+- Specificity hierarchy
+- The Box Model
+
+### Materials
+[CSS Diner](https://flukeout.github.io/)
+[Flexbox Froggy](https://flexboxfroggy.com/)
+[Flexbox Cheatsheet](https://flexbox.malven.co/)
+[Add CSS to recipe-page project](../projects/recipe-page/phase-2-css-prompt.md)
+[Article: Inline CSS](https://www.codecademy.com/articles/html-inline-styles)
+
+## Lesson
+
+CSS concepts are not hard; it's using them all together effectively and neatly on a large scale that challenges people.  Experience is key, so let's get you some.
```

Comment:
Can we add something like: You are not expected to have this all memorized at the end of the lesson. Right now just get used to looking at CSS and try to see how the CSS relates to the HTML.

Is that too simplistic? I just worry people will go down rabbit holes and feel overwhelmed

Reply:
I like it